x-pack/plugins/security_solution/public/detections/components/rules/pre_packaged_rules/translations.ts

```diff
@@ -10,15 +10,15 @@ import { i18n } from '@kbn/i18n';
 export const PRE_BUILT_TITLE = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.prePackagedRules.emptyPromptTitle',
   {
-    defaultMessage: 'Load Elastic prebuilt detection rules',
+    defaultMessage: 'Install Elastic prebuilt detection rules',
   }
 );
 
 export const PRE_BUILT_MSG = i18n.translate(
   'xpack.securitySolution.detectionEngine.rules.prePackagedRules.emptyPromptMessage',
   {
     defaultMessage:
-      'Elastic Security comes with prebuilt detection rules that run in the background and create alerts when their conditions are met. By default, all prebuilt rules except the Endpoint Security rule are disabled. You can select additional rules you want to enable.',
+      'Elastic Security comes with prebuilt detection rules that run in the background and create alerts when their conditions are met. Select which rules you want to install and enable.',
```

Comment:
Can you share the reasoning behind this change? Are prebuilt rules no longer disabled by default after you install them? Has the state of the Endpoint Security rule changed as well? Is it no longer enabled by default after you install it?

Reply:
There are indeed two rules that are enabled by default, I think it is too much specificity to mention to users here - we primarily want to guide users to install the rules. Also the Container rule might be discontinued in the future, and I don't want to change UI copy when that happens. Sorry for not adding these details earlier.

Reply:
Thanks for the additional context @approksiu and good idea to future-proof this copy. 

@jkelas I adjusted the second sentence to help with the flow. Let me know what you think:

```suggestion
      'Elastic Security comes with prebuilt detection rules that run in the background and create alerts when their conditions are met. To start using them, choose rules to install and enable.',
```

Reply:
Ok, I updated the PR, thank you @nastasha-solomon